### PR TITLE
python3-networkx: update to 2.6.3.

### PR DESCRIPTION
**Pull request by pbui**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [x] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

**Review by paper42 — COMMENTED** [review]
2.6 removes python3-decorator dependency

There are a few dependencies listed in [requirements/default.txt](https://github.com/networkx/networkx/blob/networkx-2.6.3/requirements/default.txt), the library can be imported without them, do you know if it's usable without them?

**Comment by pbui** [comment]
@paper42 Thanks for taking a look.  I have removed the `python3-decorator` dependency.

The basics of networkx work without the other packages... meaning you can create a graph, add nodes, add edges, and run various algorithms on the structures.  You will need the other packages to visualize those structures however and perhaps do some processing.

I would be fine with adding them as actual dependencies as that is what Arch Linux does:

https://archlinux.org/packages/community/any/python-networkx/

And it is what the upstream authors list as requirements.

Let me know how you think we should proceed.

File: srcpkgs/python3-networkx/template [hunk 1]
```diff
@@ -1,18 +1,17 @@
 # Template file for 'python3-networkx'
 pkgname=python3-networkx
-version=2.6.2
+version=2.6.3
 revision=1
 wrksrc="networkx-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools unzip"
-depends="python3-decorator"
```
Comment (paper42) [inline srcpkgs/python3-networkx/template:8]: Now this package doesn't depend on python3 transitively, so we need to add an explicit dependency.

File: srcpkgs/python3-networkx/template [hunk 2]
```diff
@@ -1,18 +1,17 @@
 # Template file for 'python3-networkx'
 pkgname=python3-networkx
-version=2.6.2
+version=2.6.3
 revision=1
 wrksrc="networkx-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools unzip"
```
Comment (paper42) [inline srcpkgs/python3-networkx/template:7]: The unzip dependency seems to be not be necessary since 03a764245856b205c395f4bbb379cca12455409f

**Comment by paper42** [comment]
> The basics of networkx work without the other packages... meaning you can create a graph, add nodes, add edges, and run various algorithms on the structures. You will need the other packages to visualize those structures however and perhaps do some processing.
> 
> I would be fine with adding them as actual dependencies as that is what Arch Linux does

I think we should leave it like this, thank you for looking into it.

File: srcpkgs/python3-networkx/template (hunk 2 above)
Comment (pbui) [reply srcpkgs/python3-networkx/template]: OK. I have removed this.

File: srcpkgs/python3-networkx/template (hunk 1 above)
Comment (pbui) [reply srcpkgs/python3-networkx/template:8]: Ah, right.  I've added python3 as the depends.

**Pull request merged**